sansa-ml-spark/src/main/scala/net/sansa_stack/ml/spark/clustering/algorithms/DBSCAN.scala

```diff
@@ -68,7 +68,7 @@ class DBSCAN() extends Serializable {
         pointRDD.analyze()
 
         // Perform Spatial Partitioning with QuadTree
-        pointRDD.spatialPartitioning(GridType.QUADTREE, 16)
+        pointRDD.spatialPartitioning(GridType.QUADTREE, 2)
```

Comment:
Any specific reason why you had to change to 2? Would it be better to have this as a parameter to the DBSCAN algorithm instead of having a hard-coded value? Maybe consider adding a variable which specify the value of the partitioning.

Reply:
This variable is dependent on dataset size and distribution. It requires tunning according to that. Next time, when we will merge all the clustering algorithms (with the same calling procedure), then I will add this variable into the config file.

Reply:
So, you want to push with a static value at this iteration?

Reply:
Yes, sure. As this notebook is prior importance for the paper and the next task for SLIPO. So, please go ahead! I will surely change it in the next iteration :100:

Reply:
OK. WIll merge if for now, but created an issue (#15 ) too keep track of the discussion :)